library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -139,9 +139,14 @@ public void onCreate() {
         ContentResolver contentResolver = getContentResolver();
         File downloadDataDir = StorageManager.getDownloadDataDirectory(this);
         File externalStorageDir = Environment.getExternalStorageDirectory();
+        File[] externalStorageDirs = new File[0];
+        if (android.os.Build.VERSION.SDK_INT >= 19 )
```

Comment:
@Hackmodford what happens if this is running on a pre-19 device? is it not supported?